echo_server.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import socket
+
+def response_ok():
+    return u"HTTP/1.1 200 OK\r\n \
+    Content-Type: text/plain\r\n\r\n".encode('utf-8')
```

Comment:
This is pretty nasty looking.  Using a continuation character (`\`) is much less readable than using a multi-line string, or a list of strings that you join, or pretty much any other approach.